bindings/c/test/mako/mako.c

```diff
@@ -212,17 +218,40 @@ int populate(FDBTransaction *transaction, mako_args_t *args, int worker_id,
         xacts = 0;
         timer_prev.tv_sec = timer_now.tv_sec;
         timer_prev.tv_nsec = timer_now.tv_nsec;
+
+	/* enable transaction tracing */
+	if (args->txntrace) {
```

Comment:
Does this mean that enabling tracing disables throttling? It's a little tricky to tell with the inconsistent indenting, but I wanted to make sure I'm interpreting that correctly and then check whether that behavior was intended.

Reply:
No, throttling and txn tracing are independent.  It's just sharing the same timer.  Sorry about the indentation!